projects/container_bench/testing/test.py

```
@@ -0,0 +1,124 @@
+@entrypoint(ignore_secret_path=True)
+def export_artifacts(artifacts_dirname):
+    export.export_artifacts(artifacts_dirname)
+
+
+@entrypoint(apply_preset_from_pr_args=False)
+def matbench_run():
+    """
+    Runs one test as part of a MatrixBenchmark benchmark, includuing the deployment phase
+    """
+
+    test_container_bench.matbench_run_one()
```

Pylint (3.3.7)

[error] Module 'test_container_bench' has no 'matbench_run_one' member

(E1101)